code/modules/mob/living/basic/bots/medbot/medbot.dm

```diff
@@ -246,6 +246,17 @@
 	playsound(src, SFX_SPARKS, 75, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
 	return TRUE
 
+/mob/living/basic/bot/medbot/examine()
+	. = ..()
+	if(medical_mode_flags & MEDBOT_TIPPED_MODE)
+		var/list/panic_state = list(
+		"It appears to be tipped over, and is quietly waiting for someone to set it right.",
+		"It is tipped over and requesting help.",
+		"They are tipped over and appear visibly distressed.",
+		span_warning("They are tipped over and visibly panicking!"),
+		span_warning("<b>They are freaking out from being tipped over!</b>")
+		)
+		. += pick(panic_state)
```

Comment:
```suggestion
	if(!(medical_mode_flags & MEDBOT_TIPPED_MODE))
		return
	var/static/list/panic_state = list(
		"It appears to be tipped over, and is quietly waiting for someone to set it right.",
		"It is tipped over and requesting help.",
		"They are tipped over and appear visibly distressed.",
		span_warning("They are tipped over and visibly panicking!"),
		span_warning("<b>They are freaking out from being tipped over!</b>")
	)
	. += pick(panic_state)
```

Adjust indentation 
+ Early return
+ This can be a static list